views/eApostilles/additionalReference.ejs

```diff
@@ -5,22 +5,42 @@
     <%- partial('../partials/inner-header')%>
 </div>
 
-<div class="column-two-thirds">
+    <%if (inputError) { %>
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+            data-module="govuk-error-summary">
+            <h2 class="govuk-error-summary__title" id="error-summary-title">
+                There is a problem
+            </h2>
+            <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                    <li>
+                        <a href="#user-reference">Enter a reference no longer than 30 characters</a>
```

Comment:
The suggested error message from the design system is

>Job description must be 350 characters or fewer

https://design-system.service.gov.uk/components/character-count/error/index.html

unless a content designer or research has suggested otherwise, ideally you would use what the design system have suggested. At the bottom of each of the components, usually there's some text about what the error messages should look like. Usually it's an amalgamation of things that have already been tested, researched, refined etc

Reply:
Ah, I don't know why I didn't notice that before. I pretty much copied the error from the existing service and Liz (our content writer) said it was okay. I will change it and let her know. I'm sure she'll be fine with it.